core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -408,19 +408,25 @@ public List<String> getWhiteList() {
   public BlockReader createUfsBlockReader(long sessionId, long blockId, long offset,
       boolean positionShort, Protocol.OpenUfsBlockOptions options)
       throws IOException {
+    UnderFileSystemBlockStore.BlockAccessToken block =
+        mUnderFileSystemBlockStore.acquireAccess(sessionId, blockId, options)
```

Comment:
BlockAccessToken  is just a wrapper of sessionId and blockId.  But the caller of acquireAccess has the sessionId and blockId already, why do you make acquireAccess return the sessionId and blockId back to the caller?